packages/datetime/src/timezone/timezoneItem.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import { IconName } from "@blueprintjs/core";
+
+/** Timezone-specific QueryList item */
+export interface ITimezoneItem {
+    /**
+     * The actual timezone name, like `Europe/Paris`.
+     *
+     * This property is used as the identifier for a timezone item, so it should be unique.
+     * If you can't guarantee uniqueness, the `key` property can override it.
+     */
+    timezone: string;
```

Comment:
I would probably just call this `name: string`

Reply:
this component is getting dangerously close to nothing more than a `Select` with some default props...

Reply:
i might prefer `value` to match the controlled prop name?

Reply:
I guess it adds fuzzaldrin filtering on top too?

Reply:
but do we want to include that dependency in `datetime`? i just removed it locally...